geode-web/build.gradle

```diff
@@ -51,6 +51,7 @@ dependencies {
   compileOnly project(path: ':geode-core', configuration: 'classesOutput')
   testCompile project(path: ':geode-core', configuration: 'classesOutput')
 
+  testCompile project(':geode-core')
```

Comment:
The line just above this isn’t sufficient?

Reply:
apparently no. Without "testCompile project(':geode-core')", the IDEA editor would show red lines, without "testCompile project(path: ':geode-core', configuration: 'classesOutput')", gradle build failed at compile. Looks like we need both to satisfy both worlds.